astropy/utils/tests/test_misc.py

```diff
@@ -9,6 +9,7 @@
 import numpy as np
 
 from astropy.utils import data, misc
+from astropy.utils.exceptions import AstropyDeprecationWarning
```

Comment:
This isn't used in the file, is it?

Reply:
I was gonna add a test to catch that warning, I think. I'll come back to this PR when I get the chance, unless someone beats me to it.

Reply:
I have to check if this import is still needed with the way I mark pytest below.